civihr_employee_portal/views/templates/views-view-table--Documents--block.tpl.php

```diff
@@ -155,5 +160,13 @@ function filterTable(statusId) {
     $filtersDropdown.on('change', function (e) {
       filterTable($(this).val());
     });
+
+    // Listen for ready event when T&A finishes loading all modules
+    document.addEventListener('taReady', function (e) {
```

Comment:
`taReady` event is sent by the following code in `js/src/tasks-assignments/app.js` in T&A
```js
define([
  ... all required modules
], function () {
  'use strict';
  ...
  document.dispatchEvent(typeof window.CustomEvent === 'function' ? new CustomEvent('taReady') : (function () {
    var e = document.createEvent('Event');

    e.initEvent('taReady', true, true);

    return e;
  })());
});
```
It triggers the `taReady` when all modules are loaded.